closed/custom/modules/java.base/Copy.gmk

```diff
@@ -30,6 +30,14 @@ $(call openj9_copy_files,, \
 	$(OPENJ9_VM_BUILD_DIR)/include/jvmti.h \
 	$(INCLUDE_TARGET_DIR)/jvmti.h)
 
+$(call openj9_copy_files,, \
+	$(OPENJ9_VM_BUILD_DIR)/j9cfg.h \
+	$(SUPPORT_OUTPUTDIR)/modules_include/java.base/j9cfg.h)
+
+$(call openj9_copy_files,, \
+	$(OPENJ9_VM_BUILD_DIR)/omr/omrcfg.h \
+	$(SUPPORT_OUTPUTDIR)/modules_include/java.base/omrcfg.h)
```

Comment:
These paths are incorrect for UMA builds; see https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/592#discussion_r1178328424.

Have you verified that `java.base-copy` is a prerequisite of the makefile targets for building JDK native code?

Reply:
Added a case for UMA builds, and verified it works.

Also checked that modification of `openj9/runtime/include/j9cfg.h.[in|ftl]` is updated in generated `j9cfg.h` and JDK native build, though the target dependency is not clear from the makefile script perspective.

Reply:
I suspect that the necessary dependency is in [`make/Main.gmk`](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/make/Main.gmk#L907):
```
  $(LIBS_TARGETS): $(JVM_MAIN_LIB_TARGETS)
```
This expresses the need to build the VM before building any openjdk native libraries, aided by [`closed/custom/Main.gmk`](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/closed/custom/Main.gmk#L29):
```
JVM_MAIN_LIB_TARGETS := j9vm-build
```

Reply:
The dependencies I mentioned above are not, by themselves, sufficient to guarantee a proper build order.
This also relies on the dependency in `closed/custom/Main.gmk`:
```
java.base-libs : java.base-copy
```
From my perspective, that makes this somewhat fragile, and, unfortunately, that doesn't benefit rules to build libraries for other modules. I think this should be reworked so the header files are copied by `closed/OpenJ9.gmk` and that work is complete by the time the `build-j9` target is finished.

Reply:
Created
* https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/596